controller.py

```diff
@@ -9,80 +9,107 @@ def __init__(self,path):
+                print(f"{index}: {score}")
+        
+        self.current_score = self.current_criterion.scores[int(k)]
+
+    def edit_score_select(self, k=None):
+        print(f"You selected to update: {self.current_criterion.name} with the score of {self.current_score}")
+        print("press 1 to continue with this change, or 0 to exit editing")
+        
+        self.options = {"1":self.edit_score_confirm,
+                        "0":self.edit_criteria_quit}
+              
+    def edit_score_confirm(self,k=None):
+        """Confirm changes to score"""
+        print(f"Updated score: {self.current_criterion.name}:{self.current_score} and back to (applicant details)...")
+        self.current_applicant.scores.update({self.current_criterion.name:self.current_score})
+        self.view.view_applicant_details(self.current_applicant)
+        self.options = self.options_appdetail
+    
+    def edit_criteria_quit(self, k=None):
+        """Exit editing"""
+        print(f"You selected (stop editing the current criteria). Back to (applicant details)")
+        self.options = self.options_appdetail
+
     def run(self):
 
         self.show_boot_message()
 
         while True:
             k = readkey()
+
             if k == "q":
                 print("exiting the program...")
                 break
 
-            elif self.current_view == "home":
-                options = {"r":self.show_role_info,
-                           "a":self.show_applicants_list}
-                
-            elif self.current_view == "applicants_list":
-                options = {"b":self.show_boot_message,
-                           "d":self.show_applicant_details}
-                
-            elif self.current_view == "applicant_details":
-                options = {"a":self.show_applicants_list,
-                           "b":self.show_boot_message,
-                           "O":self.open_applicant_pdf}
-
-            elif self.current_view == "role":
-                options = {"b":self.show_boot_message}
-                
-            elif self.current_view == "criteria":
-                options = {"b":self.show_boot_message}
+            if k == "?":
+                print("---List of shortcuts---")
+                print("q: Exit the program")
+                for keypress,func in self.options.items():
+                    print(f"{keypress}: {func.__doc__}")
             
-            if options is not None:
-                output = options.get(k)
-                if k == "?":
-                    print("---List of shortcuts---")
-                    print("q: Exit the program")
-                    for keypress,func in options.items():
-                        print(f"{keypress}: {func.__doc__}")
-                elif output is not None:
-                    output()
-
+            else :
+                output = self.options.get(k)
+                if output is not None:
+                    output(k=k)
```

Comment:
Add an empty newline here to add some space in the UI between different actions.